examples/fairing.rs

```diff
@@ -1,17 +1,16 @@
-#![feature(proc_macro_hygiene, decl_macro)]
-use rocket;
-use rocket_cors;
+use std::error::Error;
 
 use rocket::http::Method;
 use rocket::{get, routes};
-use rocket_cors::{AllowedHeaders, AllowedOrigins, Error};
+use rocket_cors::{AllowedHeaders, AllowedOrigins};
 
 #[get("/")]
 fn cors<'a>() -> &'a str {
     "Hello CORS"
 }
 
-fn main() -> Result<(), Error> {
+#[tokio::main]
```

Comment:
The more idiomatic way to do this is to use the `#[rocket::main]` macro to decorate an `async fn main() {}` method that starts up and awaits on a Rocket instance.

This is subject to change as Rocket gets closer to its 0.5 release, but at the moment, this is the preferred approach.

Here's an example from Rocket's `examples/` directory: https://github.com/SergioBenitez/Rocket/blob/45b4436ed3a7ab913d96c2b69ee4df7fd8c0c618/examples/errors/src/main.rs#L39-L45

Reply:
Good point, I did'nt not about these. I now replaced `#[tokio::main] -> #[rocket::main]` and `#[tokio::test] -> #[rocket::async_test]`.